releasenotes/notes/2.0/fixes_11379-4c4d6d71213a1c8c.yaml

```diff
@@ -1,5 +1,5 @@
 ---
 upgrade_circuits:
   - |
-    Converting a quantum circuit to a gate with :func:`.converters.circuit_to_instruction` fails properly when given
-    circuit contains control flow instructions.
+    Converting a quantum circuit to a gate with :func:`.converters.circuit_to_instruction`
+    now properly fails when given circuit contains control flow instructions.
```

Comment:
Should this be a fix note? It reads like a bugfix since the behavior didn't change just the error type is more accurate?